src/lib.rs

```diff
@@ -2079,7 +2085,25 @@ impl RtcConfig {
     ///
     /// This includes setting the initial estimate to start with.
     pub fn enable_bwe(mut self, initial_estimate: Option<Bitrate>) -> Self {
-        self.bwe_initial_bitrate = initial_estimate;
+        match initial_estimate {
```

Comment:
Yeah I'm not entirely happy with this, but I think the current API is a poor since the order of calling is important(enable_bwe before `enable_loss_based_bwe`). Like you say either we take the entire BweConfig here or we do something like

```rust
pub fn bwe_config(&mut self) -> &mut BweConfig {
}
```

with `BweConfig` as 

```rust
struct BweConfig {
  enabled: bool,
  loss_based_config: LossBasedBweConfig,
}
```

Then usage becomes

```rust
let rtc = todo!();
let bwe_config = rtc.bwe_config();
bwe_config.loss_based_config.enabled = true;
bwbwe_config.loss_based_config.some_other_tunable_config = 10;
```

I think I prefer this a bit because it let's str0m stay in control of the defaults for these structs and doesn't require the user to construct them

Reply:
what about `enable_bwe(initial_estimate, loss_based_enabled)`?

Reply:
Well. We're in semver 0.x and there will be breaking changes. Thinking about that, I think what you got is probably correct. Just rename `enable_loss_based_bwe` to `experimental_enable_loss_based_bwe` maybe?

Reply:
Aight